tutorials/mesh.rst

```diff
@@ -112,7 +112,7 @@ Creating the Template
 From the OpenWISP navigation menu, go to ``Configurations``
 and then ``Templates``, from here click on the ``Add template``.
 
-.. image:: ../images/mesh/create-template.png
+.. image:: ../images/create-template.png
```

Comment:
I am not convinced of not grouping images into dedicated directories. We could have a directory called `tutorials/` for any image which is used in multiple places.